pfio/v2/local.py

```diff
@@ -158,3 +158,6 @@ def glob(self, pattern: str):
         return [
             str(item.relative_to(self.cwd))
             for item in pathlib.Path(self.cwd).glob(pattern)]
+
+    def _canonical_name(self, file_path: str) -> str:
+        return "local:/" + os.path.normpath(os.path.join(self.cwd, file_path))
```

Comment:
Default scheme of local filesystem is `file:/`.

```suggestion
        return "file:/" + os.path.normpath(os.path.join(self.cwd, file_path))
```

See also:
- https://pfio.readthedocs.io/en/latest/design.html#uri-expression-of-file-paths
- `fs._from_scheme()`
- https://www.iana.org/assignments/uri-schemes/uri-schemes.xhtml

Reply:
Thank you for your review. Fixed by https://github.com/pfnet/pfio/pull/316/commits/4ad0497bd4abbbd60ca308fa99bfec3bb5d4b1e4